### PR TITLE
Add usage instructions for sdk completion

### LINKED ISSUE #40: Add usage instructions for sdk completion
The new `completion` command is currently lacking usage instructions.

### PR DESCRIPTION
**Pull request by helpermethod**
Fixes #40.

### CODE REVIEW

File: app/views/usage.scala.html [hunk 1]
```diff
@@ -203,6 +204,37 @@ <h2>Current Version(s)</h2>
...
                 </article>
                 <br/>
+                <p>
+                    <a name='completion'></a>
+                    <h2>Shell Completion</h2>
+                    <p>
+                    With SDKMAN!'s rich set of commands and subcommands, it can sometimes be hard to remember which
+                    commands are supported or which subcommands are allowed by a command.
+                    Wouldn't it be nice if the shell would autocomplete them for you?
+                    Luckily, SDKMAN! comes packaged with shell completions for both Bash and Zsh (not supported yet).
```
Comment (marc0der) [inline app/views/usage.scala.html:214]: Maybe we should just leave out ZSH for now until we have added completion.

Comment (helpermethod) [reply app/views/usage.scala.html:214]: Good point!

File: app/views/usage.scala.html [hunk 2]
```diff
@@ -203,6 +204,37 @@ <h2>Current Version(s)</h2>
...
+                    When double-tapping after entering a top-level command, the list of subcommands is shown:
+<pre><code>
+$ sdk env &lt;tab&gt;&lt;tab&gt;
+clear    init     install
+</code></pre>
+                    Note that shell completion is currently not supported for candidates and versions.
+                    <p>
+                    </p>
+                    To permanently enable shell completion, add it to your <code>.bashrc</code> (Linux)
+                    <code>.bash_profile</code> (Mac), or <code>.profile</code> (Unix), right after the
```
Comment (marc0der) [inline app/views/usage.scala.html:232]: On Unix based systems we [use the `.bashrc`](https://github.com/sdkman/sdkman-hooks/blob/1079a92c33e0cd663111b9de03cfcf50b72fc37d/app/views/install.scala.txt#L286) to initialise the bash profile.

**Comment by marc0der** [comment]
Looks great, just two minor tweaks.

**Comment by helpermethod** [comment]
I've made all requested changes.

**Comment by marc0der** [comment]
Looks great, thanks!

**Pull request merged**